website/static/specification.html

```diff
@@ -609,6 +614,7 @@ <h2><dfn>MonetizationProgressEvent</dfn> interface</h2>
               readonly attribute DOMString amount;
               readonly attribute DOMString assetCode;
               readonly attribute DOMString assetScale;
+              readonly attribute DOMString? receipt;
```

Comment:
Should there be a question mark here? I don't know what the right format is :)

Reply:
The `?` is intentional since the receipt is optional.
That said I'm also not super familiar with this formatting.

Reply:
@adrianhopebailie would likely know this the best

Reply:
There's also references to the Web-Monetization-Id header in the spec. That was removed from the other Web Mon doc. Should that come out?

Reply:
Thanks. Removed in https://github.com/interledger/webmonetization.org/pull/51/commits/10a5fd13c2add0f347ba5e56a83fb87503998125

Reply:
From my understanding of WebIDL @wilsonian has it right